---
title: biochemistry II
date: 2022-12-18
---

## chapter 13 Bioenergetics and biochemical reaction types

Recall basic formulas for chemical energy such as free energy G:
$$
ΔG = ΔH-TΔS
$$
catabolism represents the reactions that release energy, and anabolism includes reactions that needs energy.

# Chapter 25 DNA metabolism

The deduced double helix structure indicated indicated that DNA has a semiconservative mechanism of replication. This is proved by a famous experiment conducted in *E. coli* cells. After labeling the DNA with $^{15}N$ which makes the chain heavier, they found that the newly synthesized DNA has H-H, H-L and L-L. Yet this is a rough understanding on DNA replication. Some details remains to be determined.

Deepgoing study is carried out with autoadiography. As for *E. coli* whose chromosomal DNA is a huge circle, daughter strands are synthesized immediately after parental strands separate. No temporary creation of linear DNA occurred during replication of the circular DNA. DNA replication begin at specific sites and proceed bidirectionally. After this, the researchers purified the DNA polymerase and studied DNA polymerization in vitro.

Basic information is that when DNA polymerase works in vitro, it needs DNA template, primer and dNTPs. The fundamental reaction for DNA synthesis is a nucleophilic attack by the 3'-OH group of the strand on the 5' α-phosphorus of a incoming dNTP selected via base-pairing. 

DNases can be divided into exonuclease and endonuclease. The former remove the nucleotide one by one, either from 5' to 3' or from 3' to 5', while the latter can cut the DNA at any site. DNA pol I can be divided into two fragments.

![Quicker_20221218_134123](https://s2.loli.net/2022/12/19/ZvoWNmAeaptjED9.png)

In DNA pol I, 3'->5' exonuclease activity is able to remove mismatched base pairs, in order to proofread the nucleotides, and 5' -> 3' enables it to catalyze the nick translation process which removes RNA primers and can be used in DNA repair.

The synthesis of daughter DNA strands is semidiscontinuous. One of the strands  (called leading strand) is synthesized continuously and the other one (called the lagging strand) discontinuously. Okazaki fragments occurs on the lagging strand. These Okazaki fragments will be join together by DNA lingase to form a complete strand. 

Despite all these discussions about DNA pol I, this enzyme is found not to be responsible for DNA replication. Its velocity is too low and it can only add about 50 nucleotide before it dissociates from the template. The one that is responsible for DNA replication is actually DNA pol III which have no 5' ->3' exonuclease activity. 

DNA pol III has three main subunits. α subunit has polymerization activity, ε subunit has 3' -> 5' proofreading exonuclease activity, and β subunit has clamp activity that binds the whole enzyme tightly to the DNA.

Aside from DNA polymerase III, many other proteins are also part of the **replisome** for DNA replication in *E. coli*, such as

1. the helicase (dnaB) that consumes ATP to separate the two strands.
2. the topoisomerases that relieve the topological strains in the helical structure.
3. the single strand binding protein (SSB) that binds to the single-stranded DNA generated.
4. the primase (dnaG) that generates a short RNA primer.
5. the DNA polymerase I (polA) that removes the primer and replace it by a DNA sequence.
6. the DNA ligase for sealing nicks between the Okazaki fragments.

In *E. coli*, the replication starts from a 245bp fragment called *OriC*. *OriC* contains 11 GATC sequences, the A can be methylated, replication is conducted once a time. 

The whole process of *E. coli* chromosomal DNA replication has three stages: initiation, elongation and termination. Various enzymes are needed in initiation, and the synthesis of leading and lagging strand is believed to be conducted by a  DNA pol III dimer. During termination stage, the two replication forks (moving in opposite direction) meet at the Ter sequence, and DNA topoisomerase IV separated the newly generated chromosome with the old one.

**proteins required in initiation**  

| **Protein**                                | $M_r$   | **Number of subunits** | **Function**                                                 |
| ------------------------------------------ | ------- | ---------------------- | ------------------------------------------------------------ |
| DnaA protein                               | 52,000  | 1                      | Recognizes *oriC*  sequence;  opens duplex at specific sites in origin |
| DnaB protein (helicase)                    | 300,000 | 6                      | Unwinds DNA                                                  |
| DnaC protein                               | 174,000 | 6                      | Required for DnaB binding at  origin                         |
| HU                                         | 19,000  | 2                      | Histonelike protein; DNA-binding  protein; stimulates initiation |
| FIS                                        | 22,500  | 2                      | DNA-binding protein; stimulates  initiation                  |
| IHF                                        | 22,000  | 2                      | DNA-binding protein; stimulates  initiation                  |
| Primase (DnaG protein)                     | 60,000  | 1                      | Synthesizes RNA primers                                      |
| Single-stranded DNA-binding  protein (SSB) | 75,000  | 4                      | Binds single-stranded DNA                                    |
| DNA gyrase (DNA topoisomerase II)          | 400,000 | 4                      | Relieves torsional strain  generated by DNA unwinding        |
| Dam methylase                              | 32,000  | 1                      | Methylates (5')GATC sequences at *oriC*                      |

![image-20221219142341438](https://s2.loli.net/2022/12/19/UjMOaDk3CsE6otl.png)

![image-20221219144551418](https://s2.loli.net/2022/12/19/1BqwhevaYSORLm6.png)

![image-20221219143916273](https://s2.loli.net/2022/12/19/WeyQ4p8bLFElmjU.png)

![image-20221219144848907](https://s2.loli.net/2022/12/19/rMugWiYRdI58oqf.png)

# Chapter 26 RNA metabolism

There are different types of RNA, some important types are rRNA (ribosome RNA), tRNA (transfer RNA) and mRNA (messanger RNA). RNA can also act as enzymes, there is a hypothesis that RNA catalyzes most of the metabolic reactions before protein occurred. 

Three types of RNA polymerase catalyze the transcription of most RNA.

| Type | Localization | Transcript          | Effect towards α-amanitin       |
| ---- | ------------ | ------------------- | ------------------------------- |
| I    | nucleolus    | 18S, 5.8S, 28S rRNA | insecsitive                     |
| II   | nucleopalsm  | mRNA, snRNA         | strongly sensitive              |
| III  | nucleopalsm  | tRNA, 5S rRNA       | inhibited by high concentration |

![image-20221219211016753](https://s2.loli.net/2022/12/19/Z39zkA4pWcSsTgR.png)

The mRNA that just come through transcription will be  further processed before it goes into translation. The further process include 5' end capping, removing introns (this is discovered by an interesting EM experiment), linking exons and 3' end polyadenylating. tRNA will also be cleaved, spliced and modified.

![image-20221219222627944](https://s2.loli.net/2022/12/19/79lTUaRpiqDAsd6.png)

There are four types of introns that are spliced through different mechanisms. A single mRNA can be spliced differently to encode different protein.

A single pre-rRNA can generate different types of mature RNA in prokaryotic and eukaryotic cells.

![image-20221219223416723](https://s2.loli.net/2022/12/19/OZ9Bos5r7nQIWRf.png)

![image-20221219223639922](https://s2.loli.net/2022/12/19/CPq1IneEglGdSL4.png)

Celluar 
